Services/DataX.Flow/Flow.InteractiveQueryService/FlowInteractiveQueryServiceStartup.cs

```diff
@@ -6,12 +6,15 @@ namespace Flow.InteractiveQueryService
 {
     using Microsoft.AspNetCore.Hosting;
     using DataX.ServiceHost.AspNetCore;
+    using DataX.ServiceHost.AspNetCore.Startup;
+    using DataX.Contract.Settings;
 
-    public class Startup : StartUpBase
+    public class FlowInteractiveQueryServiceStartup : DataXServiceStartup
```

Comment:
Can you add comment headers for the new class? ... and in general for all new methods that you have authored in this PR. Thanks!